src/main/java/org/truffleruby/RubyContext.java

```diff
@@ -179,7 +181,8 @@ public RubyContext(RubyLanguage language, TruffleLanguage.Env env) {
 
         Launcher.printTruffleTimeMetric("before-create-native-platform");
         nativePlatform = NativePlatformFactory.createPlatform(this);
-        featureLoader.initialize(nativePlatform);
+        truffleNFIPlatform = options.NATIVE_INTERRUPT ? new TruffleNFIPlatform(this) : null;
```

Comment:
Mmh, this check is incorrect, we use NFI for other purposes now.
I'll fix in a separate PR.